winit-core/src/window.rs

```diff
@@ -792,6 +792,16 @@ pub trait Window: AsAny + Send + Sync + fmt::Debug {
     /// ```
     fn safe_area(&self) -> PhysicalInsets<u32>;
 
+    /// The minimum dimensions of the window's surface if it was set.
+    ///
+    /// Getter for [`set_min_surface_size`][Window::set_min_surface_size], see that for details.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **iOS / Android / Orbital:** Unsupported.
+    /// - **Web:** Unimplemented, returns `None`.
```

Comment:
```suggestion
    /// - **Web:** Unimplemented.
```
I propose if it returns `Option`, we don't have to specify that it returns `None`.